Spigot-Server-Patches/0169-Optimise-NetworkManager.patch

```diff
@@ -1,88 +0,0 @@
-From fa201a9b717a5ae06c7572fcaa0811eb210d8a00 Mon Sep 17 00:00:00 2001
```

Comment:
Please explain removal of this patch. 

I would rather us keep the optimizations provided if possible. The Xray patch should extend the code changed by this patch instead.

Reply:
Note - please reply on the thread instead of as a reply to this review line as I did add some comments in the main PR - I think I now understand why - but welcome to hear if you think there are any improvements the original change included that could be retained.